### PR TITLE
Abbadie 111 others

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New record, corrections

### CODE REVIEW

File: ParisBNF/abb/BNFabb111.xml [hunk 1]
```diff
@@ -0,0 +1,689 @@
...
+
+                               <locus from="103ra" to="110rb"/>
+                               <title type="complete" ref="LIT1385Fekkar"></title>
+                               <textLang mainLang="gez"/>
+                                <incipit xml:lang="gez"><hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡</hi> አሐዱ፡
+                              አምላክ። ወዝንቱ፡ ነገር፡ ወፍካሪ፡ ኢየሱስ፡ ወተስእልዎ፡ አርዳኢሁ፡ ወእንዘ፡ ይመስሐ፡ ምስለሆሙ፡ ወሐተትዎ፡
+                            ወይቤልዎ፡ እስኩ፡ ንግረኒ፡ ኦሊቅነ፡ በደኅሪ፡ ትውልድ፡ ዘይመጽእ፡ መዋዕል፡ ምንተ፡ ይከውን፡ ተእምርቱ። ወአውሦሙ፡
+                               </incipit>
+                               <note>Cp. <bibl>
+                                  <ptr target="bm:Nosnitsin2005EAEFekkareI"/></bibl>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb111.xml:538]: is the bibliography for the ms or for the work?

File: ParisBNF/abb/BNFabb111.xml [hunk 2]
```diff
@@ -0,0 +1,689 @@
...
+                                <extent>
+                                    <measure unit="leaf">8</measure>
+                                    <locus from="103r" to="110v"/>
+                                    <dimensions type="outer" unit="mm">
+                                      <height cert="low">225</height>
+                                      <width cert="low">175</width>
+                                    </dimensions>
+                                </extent>
+
+                                  <condition key="good">One leaf (<ref target="#104"/>) have the bottom outer corner missing,
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb111.xml:560]: has

File: ParisBNF/abb/BNFabb111.xml [hunk 3]
```diff
@@ -0,0 +1,689 @@
...
+                               the name seems to have been added later, over an erasure).</note>
+                          </item>
+                       </list>
+                      </additions>
+
+
+                    </physDesc>
+                    <history>
+                        <origin>
+                     <origDate notBefore="1400" notAfter="1499" cert="medium"/>. <ref target="#p2"/>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb111.xml:625]: delete . (fullstop)

File: ParisBNF/abb/BNFabb111.xml [hunk 4]
```diff
@@ -0,0 +1,689 @@
...
+                     might be of comparable age or even somewhat older than <ref target="#p1"/>.
+                     <ref target="#a6"/> is or more recent time and seems to mention the same owner,
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb111.xml:627]: is of more recent...

File: ParisBNF/abb/BNFabb111.xml [hunk 5]
```diff
@@ -0,0 +1,689 @@
...
+                     <persName ref="PRS13365Amlakawi" role="owner">ʾAmlākāwi</persName>. It is not clear wheather
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb111.xml:628]: whether

File: ParisBNF/abb/BNFabb111.xml [hunk 6]
```diff
@@ -0,0 +1,689 @@
...
+                     <ref target="#p2"/> had a part of a disperced manuscript or a separate circulation unit.
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb111.xml:629]: was a part?

dispersed

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: ParisBNF/abb/BNFabb111.xml [hunk 7]
```diff
@@ -0,0 +1,689 @@
...
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0303BNF"/>
+                        <collection>Manuscrits orientaux</collection>
+                        <collection>Fonds éthiopien</collection>
+                        <collection>d'Abbadie</collection>
+                        <idno facs="https://gallica.bnf.fr/ark:/12148/btv1b531387732">BnF Éthiopien d'Abbadie 111</idno>
+                          </msIdentifier>
+                          <physDesc>
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb111.xml:34]: done

File: ParisBNF/abb/BNFabb111.xml [hunk 8]
```diff
@@ -0,0 +1,689 @@
...
+
+                                     <handNote xml:id="h2" script="Ethiopic" corresp="#p1">
+                                       <locus target="#36va">The Postscript of the  Gospel of John.</locus>
+                                        <seg type="script" cert="medium">18th-century script.</seg>
+                                        <seg type="ink">Black.</seg>
+                                           <desc>A careful secondary hand
+                                             that added <ref type="work" corresp="LIT1693John#Postscript"/>.</desc>
+                                     </handNote>
+
+                                     <handNote xml:id="h3" script="Ethiopic" corresp="#p1_a1">
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb111.xml:314]: ok

File: ParisBNF/abb/BNFabb111.xml [hunk 9]
```diff
@@ -0,0 +1,689 @@
...
+                                       <locus target="#37v"></locus>
+                                       <desc>Miniature of the <persName ref="PRS1962Anthony"/>,
+                                       with a rubricated legend <foreign xml:lang="gez">ሥዕለ፡ አባ፡ እንጦንስ፡</foreign>, protexted
+                                       by a textile curtain.</desc>
+                                     </decoNote>
+
+
+                                     <decoNote type="ornamentation" xml:id="d3">
+                                       <locus target="#44vb"></locus>
+                                       <desc>A fine monochrom (red) ornamental band.</desc>
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb111.xml:378]: ok

File: ParisBNF/abb/BNFabb111.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb111.xml:625]: done

File: ParisBNF/abb/BNFabb111.xml (hunk 6 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb111.xml:629]: yes

File: ParisBNF/abb/BNFabb111.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb111.xml:538]: it mentions the ms.

File: ParisBNF/abb/BNFabb111.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb111.xml:560]: ok

**Review by eu-genia — APPROVED** [review]

File: ParisBNF/abb/BNFabb111.xml [hunk 10]
```diff
@@ -0,0 +1,689 @@
...
+                                    <item xml:id="a1">
+                                      <locus from="36va" to="36vb"></locus>
+                                       <desc type="Genealogy"/>
+                                       <q xml:lang="gez">በነገረ፡ ልደትነ፡ ወቁርጸት፡
+                                         ዳውጃሆሙ፡ ወይዘሮ፡ ትኩብት፡ ወለደ፡ አርኬ፡ ነገደ፡
+                                         ኢየሱስ፡ ወማርያም፡ ክብራ። ፫አኃው፡ አርኪ፡ ቢውልድ፡
+                                         ማሙች፡ ወማሙች፡ ብትወልድ፡ ቢሌለዋ፡ ወሌለዋ፡ ብትወልድ፡
+                                         ደጅአዝማች፡ ኤሎስ፡ ወደጅ፡ አዝማች፡ ኤሎሎስ፡
+                                          ወለዱ፡ እሜት፡ ደባርዋ፡ ወምቤት፡ ደባርዋ፡
+                                          ወለዱ፡ አቤቶ፡ <pb n="36vb"/> ኤሎስ። ነገደ፡ ኢየሱስ፡ ቢወልድ፡
```
[the remark this answers is not in the document]
Comment (eu-genia) [reply ParisBNF/abb/BNFabb111.xml:438]: pb has not been changed

[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb111.xml:438]: sorry, now done.

**Review by eu-genia — APPROVED** [review]

**Pull request merged**